Cargo.toml

```diff
@@ -1,37 +1,34 @@
 [package]
 name = "tracing-axiom"
 version = "0.4.0"
-authors = ["Arne Bahlo <arne@axiom.co>"]
+authors = ["Arne Bahlo <arne@axiom.co>", "Darach Ennis <darach@gmail.com>"]
 edition = "2021"
-rust-version = "1.60"
-license = "MIT OR Apache-2.0"
+rust-version = "1.73"
+license = "LicenseRef-Axiom-Dual-ASL2-MIT"
```

Comment:
How does this work?

Reply:
I'll set it back to the expression - its more common and both licenses are well known.